uco-core/core.ttl

```diff
@@ -114,30 +114,15 @@ core:EnclosingCompilation
 		[
 			a owl:Restriction ;
 			owl:onProperty core:description ;
-			owl:maxCardinality "0"^^xsd:nonNegativeInteger ;
-		] ,
-		[
-			a owl:Restriction ;
```

Comment:
It looks like you accidentally removed the cardinality restriction for name here. 
It should still be present but adjusted to maxCardinality like description is.

Please put it back in and commit it so that I can approve.

Reply:
I did remove it, I could not see an error in the syntax, yet it was the cause of an error. I found with it for sometime. Perhaps on a night's sleep, I'll have better luck. 

Can you approve it as is, and we create another separate Issue to add the name and cardinality back?

Reply:
looks like the issue may be that it is using owl:onClass xsd:string ; when it should be using owl:onDataRange xsd:string ;?

Reply:
I don't think so, I thought I addressed all the owl:onClass xsd:string ;  errors.
But I'll look again.

Reply:
Ah... **_UcoObject has name defined_** which this inherits... that's why it should not be here.  
@sbarnum

Reply:
Ok. It is already restricted at UcoObject so restriction at EnclosingCompilation is not necessary.
I can live with that. :-)